src/Components/RelatedCollectionsRail/RelatedCollectionEntity.tsx

```diff
@@ -36,9 +36,11 @@ export class RelatedCollectionEntity extends React.Component<CollectionProps> {
       title,
     } = this.props.collection
     const artworks = artworks_connection.edges.map(({ node }) => node)
-    const bgImages = compact(artworks.map(({ image }) => image && image.url))
+    const bgImages = compact(
+      artworks.map(({ image }) => image && image.resized.url)
```

Comment:
If resized is null, this code will fail. Need to guard, similar to @pepopowitz 's comment up above.